src/components/image-container/image-container.scss

```diff
@@ -19,6 +19,23 @@
 	border: 0;
 	height: auto;
 	margin: 0;
+	object-fit: cover;
+	object-position: center;
 	padding: 0;
 	width: 100%;
 }
+
+.image-container__image--portrait {
+	display: block;
+
+	@media (min-aspect-ratio: 5/6) {
```

Comment:
I'm wondering why use `min-aspect-ratio: 5/6` instead of `orientation: portrait` ? or at least `1/1` if you could remind me ?